openmetadata-ui/src/main/resources/ui/src/utils/BotsUtils.ts

```diff
@@ -11,16 +11,14 @@
  *  limitations under the License.
  */
 
+import { t } from 'i18next';
```

Comment:
I would recommend using hook if we are using this in functional components.

Reply:
this is util file!